### PR TITLE
[SYCLomatic]Multi-process runing the test suite in the SYCLomatic-test

### PR DESCRIPTION
**Pull request by ShengchenJ**

### CODE REVIEW

**Review by zhiweij1 — APPROVED** [review]

File: run_test.py [hunk 1]
```diff
@@ -213,6 +212,7 @@ def run_test_driver(module):
         test_config.test_status = "MIGFAIL"
 
     if ret_val:
+        call_subprocess("sycl-ls")
```
Comment (tangjj11) [inline run_test.py:215]: Seem like no effect?

File: run_test.py [hunk 2]
```diff
@@ -290,7 +287,16 @@ def test_single_case(current_test, single_case_config, workspace, module, suite_
...
     copy_source_to_ws(single_case_config.test_dep_files, case_workspace, suite_root_path)
-    return run_test_driver(module)
+    test_result = run_test_driver(module)
+    # Write the execution command to command.tst, execution log to <test case>.lf, execution result to result.md
+    write_log_to_file(test_config.log_file, "------------------------------------------------------------------------\n\n" + \
+                "=================== "+ test_config.current_test + " is " + test_config.test_status + " ======================\n " + test_config.execution_log)
+    append_log_to_file(test_config.result_text, test_config.current_test + " " + test_config.test_status + "\n")
+    append_log_to_file(test_config.command_file, test_config.executed_command + "\n")
+
+    if not test_result: # When execution failed, then print the failed log in the terminal.
```
Comment (tangjj11) [inline run_test.py:297]: Print both pass case name and failed case log is better.

**Review by tangjj11 — APPROVED** [review]

**Review by zhiweij1 — COMMENTED** [review]
Could you print the compile command if there is compile fail?

**Comment by ShengchenJ** [comment]
> Could you print the compile command if there is compile fail?

Done

**Pull request merged**